week-2/multiproc/main.py

```diff
@@ -0,0 +1,59 @@
+import time
+
+from utils import predict, prepare_input
+from multiprocessing import Pool
+
+n_chunks = 20
+n_jobs = 8
+multiplier = 2
```

Comment:
would be nice to make them input params with https://typer.tiangolo.com/ or https://docs.python.org/3/library/argparse.html